packages/extension-polkagate/src/popup/staking/pool-new/Info.tsx

```
@@ -0,0 +1,124 @@
+  }, [t]);
+
+  const stakingStats = useMemo(() => ([
+    { label: t('Min {{token}} to join a pool', { replace: { token: token ?? '' } }), value: stakingInfo.poolStakingConsts?.minJoinBond },
+    { label: t('Min {{token}} to create a pool', { replace: { token: token ?? '' } }), value: stakingInfo.poolStakingConsts?.minCreationBond },
+    { label: t('Number of existing pools'), value: stakingInfo.poolStakingConsts?.lastPoolId.toString() },
+    { label: t('Max possible pools'), value: getValue(stakingInfo.poolStakingConsts?.maxPools) },
+    { label: t('Max possible pool members'), value: getValue(stakingInfo.poolStakingConsts?.maxPoolMembers) },
+    { label: t('Max pool members per pool'), value: getValue(stakingInfo.poolStakingConsts?.maxPoolMembersPerPool) }
+  ]), [getValue, stakingInfo.poolStakingConsts?.lastPoolId, stakingInfo.poolStakingConsts?.maxPoolMembers, stakingInfo.poolStakingConsts?.maxPoolMembersPerPool, stakingInfo.poolStakingConsts?.maxPools, stakingInfo.poolStakingConsts?.minCreationBond, stakingInfo.poolStakingConsts?.minJoinBond, t, token]);
+
+  const onBack = useCallback(() => navigate('/pool/' + genesisHash) as void, [genesisHash, navigate]);
```

Biome (1.9.4)

[error] void is confusing outside a return type or a type parameter.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)